control-plane-operator/controllers/hostedcontrolplane/scheduler/config_test.go

```diff
@@ -0,0 +1,60 @@
+package scheduler
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+	"time"
+
+        . "github.com/onsi/gomega"
+	"github.com/openshift/hypershift/support/config"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	componentbasev1 "k8s.io/component-base/config/v1alpha1"
+	schedulerv1beta2 "k8s.io/kube-scheduler/config/v1beta2"
+	"k8s.io/utils/pointer"
+)
+
+func TestGenerateConfig(t *testing.T) {
+	var errors []error
+	leaseDuration, err := time.ParseDuration(config.RecommendedLeaseDuration)
+	errors = append(errors, err)
+	renewDeadline, err := time.ParseDuration(config.RecommendedRenewDeadline)
+	errors = append(errors, err)
+	retryPeriod, err := time.ParseDuration(config.RecommendedRetryPeriod)
+	errors = append(errors, err)
+	for _, item := range errors {
+		g := NewWithT(t)
```

Comment:
There's no need to generate a new g every loop iteration, so that should be moved out.
Also I'd suggest to add a `g.Expect(err).ShouldNot(HaveOccurred())` after each error instead of the loop. It's simpler and easier to read.